src/main/resources/swagger/ga4gh-tool-discovery.yaml

```diff
@@ -235,6 +235,38 @@ paths:
           description: The tool can not be output in the specified type.
           schema:
             $ref: '#/definitions/Error'
+  /tools/{id}/versions/{version_id}/{type}/files:
+    get:
+      summary: Get an array of objects that contain the relative path and file type.  Intended for use with the /tools/{id}/versions/{version_id}/{type}/descriptor/{relative_path} endpoint.
+      tags:
+        - GA4GH
+      parameters:
+        - name: type
+          required: true
+          in: path
+          description: The output type of the descriptor. If not specified it is up to the underlying implementation to determine which output type to return. Plain types return the bare descriptor while the "non-plain" types return a descriptor wrapped with metadata. Allowable values are "CWL" and "WDL".
```

Comment:
This is returning an array of ToolFile, so this description (especially the part about plain types) seems off

Reply:
Changing...